app/components/hivers/template.html

```diff
@@ -1,8 +1,8 @@
-<div ng-controller="HiversCtrl">
+<div class="container">
   <h4>Hivers</h4>
   <span class="list-group">
     <span class="col-sm-3 ph-sm-padding"
```

Comment:
Can you change this to `col-sm-4`? Some elements are now squeezed a bit with the introduced `container` class (which definitely improves the presentation!).
![screenshot from 2016-03-28 18-11-13](https://cloud.githubusercontent.com/assets/1874116/14082498/9f7f8c44-f510-11e5-810a-d40f2db5712a.png)